jitpack.yml

```diff
@@ -0,0 +1,5 @@
+before_install:
+  - sdk install java 8.0.422-tem
+  - sdk use java 8.0.422-tem
+install:
+  - ./gradlew PskipSigning=true -Ptempto_group=com.github.aaneja.tempto clean publishToMavenLocal
```

Comment:
```suggestion
  - ./gradlew -PskipSigning=true -Ptempto_group=io.prestodb.tempto clean publishToMavenLocal
```

Reply:
Actually, tempto_group should probably be `com.github.prestodb.tempto`?

Reply:
I wanted to use my own groupid for testing. Since this is jitpack related, it will have no bearing on the actual release artifacts

Reply:
I will delete this file